.travis.yml

```
@@ -10,9 +10,6 @@ language: rust
 rust:
   - 1.29.2
```

Needs to be replaced with `stable`:
```suggestion
  - stable
```